pytest_django/fixtures.py

```diff
@@ -33,29 +33,25 @@
 
 
 @pytest.fixture(scope="session")
-def django_db_modify_db_settings_tox_suffix():
+def django_db_modify_db_settings_suffix():
     skip_if_no_django()
-
+    xdist_worker = getattr(request.config, "slaveinput", {}).get("slaveid")
     tox_environment = os.getenv("TOX_PARALLEL_ENV")
+    suffix_parts = []
     if tox_environment:
         # Put a suffix like _py27-django21 on tox workers
-        _set_suffix_to_test_databases(suffix=tox_environment)
-
-
-@pytest.fixture(scope="session")
-def django_db_modify_db_settings_xdist_suffix(request):
-    skip_if_no_django()
-
-    xdist_suffix = getattr(request.config, "slaveinput", {}).get("slaveid")
-    if xdist_suffix:
+        suffix_parts.append(tox_environment)
+    if xdist_worker:
         # Put a suffix like _gw0, _gw1 etc on xdist processes
-        _set_suffix_to_test_databases(suffix=xdist_suffix)
+        suffix_parts.append(tox_environment)
+    suffix = "_".join([tox_environment, xdist_worker])
+    if suffix:
+        _set_suffix_to_test_databases(suffix=suffix)
```

Comment:
The idea here was to allow for e.g. having your own/no suffix with tox, by defining the `django_db_modify_db_settings_tox_suffix` fixture yourself.
And it appears to be another case for hding an issue with pytest 5.3.3, where the order of fixture appears to have become flaky in this regard then, no?

Reply:
Ah, I wasn't aware. I have another solution in mind. I'll push it really quick.

Reply:
I also just realized I missed the import stuff, which would have also been a problem